core/src/main/scala/zio/jdbc/Sql.scala

```diff
@@ -210,6 +221,15 @@ object Sql {
     implicit val sqlDateSetter: Setter[java.sql.Date] = forSqlType((ps, i, value) => ps.setDate(i, value), Types.DATE)
     implicit val sqlTimeSetter: Setter[java.sql.Time] = forSqlType((ps, i, value) => ps.setTime(i, value), Types.TIME)
 
+    implicit def sqlIterableSetter[A](implicit setter: Setter[A]): Setter[Seq[A]] =
```

Comment:
I think due to contravariant type classes being broken in Scala 2.x, the invariance means that we need to have concrete classes:

```scala
Setter[Chunk[A]]
Setter[List[A]]
Setter[Vector[A]]
Setter[Set[A]]
Setter[Array[A]]
```

Probably those types are enough.